nuitka/importing/Importing.py

```diff
@@ -52,7 +51,6 @@
 from .PreloadedPackages import getPreloadedPackagePath, isPreloadedPackagePath
 from .Whitelisting import isWhiteListedNotExistingModule
 
-_debug_module_finding = Options.shallExplainImports()
```

Comment:
Agree, this was a misguided attempt at preserving looks of old code when I added that option.

Reply:
the only reason why got rid of it was to decouple Options, so don't get me wrong... sometimes globals on modules can serve the purpose. Think about this PR getting merged as an intermediate step for more PRs completing this one. Right now I just want to be able to delete the branch locally so I can work on a more clean PR with these minor changes living on upstream